README.md

```diff
@@ -12,117 +12,82 @@
 
 ## Installation
 
-The `linearham` installation dependencies are specified in the `Dockerfile` file.
-To install the dependencies yourself, you must clone this repository and then run each command in the `Dockerfile` that comes after `RUN`.
-Alternatively, you may use Docker (see below).
+Linearham's dependencies are described in the Dockerfile.
+We recommend that you run linearham inside a Docker container, since this will make installation much easier (if you're new to Docker, [read this](http://erick.matsen.org/2018/04/19/docker.html)).
+However, you can also install the dependencies by hand, in which case you should clone the repository and run each command in the [Dockerfile](https://github.com/matsengrp/linearham/blob/edit-readme/Dockerfile) that's on a line starting with `RUN` (treat `WORKDIR` as `cd`).
+The more similar your system is to that described by the Dockerfile's `FROM` line (at the moment, debian), the easier this will be.
 
-Note that the `Dockerfile` specifies build instructions for computers running Debian operating systems.
-The installation steps have been successfully tested using the Docker image, which is based on Debian version 9 AKA "stretch."
-We expect the installation process to work on similar UNIX-based platforms.
+## Using Docker
 
-## Docker
+It's best to start by running an interactive session in the container:
 
-You can run Linearham using [our Docker image on quay.io](https://quay.io/repository/matsengrp/linearham); if you are new to Docker, [read this](http://erick.matsen.org/2018/04/19/docker.html#pulling-and-running-a-docker-image).
+`docker run -it quay.io/matsengrp/linearham /bin/bash`.
 
-Before running with Docker, choose an image tag to use from [those published to quay](https://quay.io/repository/matsengrp/linearham?tab=tags).
-We recommended you use the most recent **version-specific** tag (e.g. `v1.0.0`, not `latest`) in order to allow for reproducibility.
+You can also create a shell script with your linearham commands for docker to run, and put it in place of `/bin/bash` (use `test.sh` as an example).
+If you want your run to be reproducible, choose a tag of the form vX.X.X [from quay.io](https://quay.io/repository/matsengrp/linearham?tab=tags), then specify it like so: `quay.io/matsengrp/linearham:vX.X.X`.
```

Comment:
@psathyrella many of the tags on [quay.io](https://quay.io/repository/matsengrp/linearham?tab=tags) are more like `v1.0.0-21-g1df75c2`. Do you think the way you have it here will discourage the use of tags that include a number of commits / hash as part of the tag name (there's no reason why people should not use images with tags that look like that) in addition to just the git tag version? 

Or maybe we should just tag our images according to `git describe --tags --abbrev=0` aka just the git tag and not include the number of commits after that tag / the hash.

Reply:
i guess i was reading how it was before to be saying to avoid the bleeding edge latest one, as well as for reproducibility. But if we just want people to be able to reproduce their builds, can't they just look at the tag it was build with afterwards based on the image it downloaded? In which case they can always just pull latest, unless they want to go back and recreate something.